tests/uefi_sb/utils.py

```diff
@@ -84,7 +84,7 @@ def sign_efi_bins(vm, db):
     if shutdown:
         vm.shutdown(verify=True)
 
-def _test_key_exchanges(vm):
+def test_key_exchanges(vm):
```

Comment:
The underscore was here to protect the function from unwanted collection by pytest:

Before:
```
 $ pytest --collect-only --hosts=host tests/uefi_sb/ -q | grep test_key_exchanges
tests/uefi_sb/test_uefistored_sb.py::TestUEFIKeyExchange::test_key_exchanges[None]
tests/uefi_sb/test_varstored_sb.py::TestUEFIKeyExchange::test_key_exchanges[None]
```

After:
```
$ pytest --collect-only --hosts=host tests/uefi_sb/ -q | grep test_key_exchanges                                                                                        
tests/uefi_sb/test_uefistored_sb.py::test_key_exchanges                                                                                                                                                                               
tests/uefi_sb/test_uefistored_sb.py::TestUEFIKeyExchange::test_key_exchanges[None]                                                                                                                                                    
tests/uefi_sb/test_varstored_sb.py::test_key_exchanges                                                                                                                                                                                
tests/uefi_sb/test_varstored_sb.py::TestUEFIKeyExchange::test_key_exchanges[None]
```